### PR TITLE
Update a few packages' version

### PR DESCRIPTION
**Pull request by fayejf**

### CODE REVIEW

File: requirements/requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 numpy>=1.18.2
 onnx>=1.7.0
 python-dateutil
-torch>1.7
+torch>=1.10
```
Comment (titu1994) [inline requirements/requirements.txt:4]: This shouldnt be there, it will override container version of torch

**Review by titu1994 — CHANGES_REQUESTED** [review]
Overall looks great, but revert the PT >= 1.10 for now, we will update it in the future relese after subsequent 21.11 comes out.

**Review by titu1994 — DISMISSED** [review]
LGTM, @ericharper for final approval before merge.

**Review by ericharper — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**